task_DEV_6/task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+
+namespace task_DEV_6
+{
+    class CustomStack
+    {
+        private int[] arrayStack;
+        private int count;
+        private const int ARRAYLENGTH = 10;
+
+        public CustomStack()
+        {
+            arrayStack = new int[ARRAYLENGTH];
+        }
+
+        public CustomStack(int item)
+        {
+            arrayStack = new int[item];
+        }
+
+        public int Capacity()
+        {
+            return arrayStack.Length;
+        }
+
+        public int Count()
+        {
+            return count;
+        }
+
+        public bool IsEmpty()
```

Comment:
In methods like this more preferable to use direct logic. Smth like: return true always except cases when count is not 0. It is not mistake just advice for the future.